sourcegraph/about
# Sticky navbar and improvements

Closes #5401 and implements a sticky nav feature.

### Changelog
- Fixed a bug correcting the right click event button. Clicking our logo now properly downloads our brand kit 🥚
- Updated our mobile navbar button/hamburger menu to match our DLS
- Implemented sticky nav functionality so when scrolling down it sticks

### Test

1. Ensure prettier has standardized the proposed changes.
2. Right click our logo to ensure the brand kit downloads
3. Resize the browser to ensure our mobile nav toggle button displays on mobile. Check light/dark version (ie: use cases page)
4. Scroll down and ensure our navbar is fixed while scrolling
